java-frontend/src/main/java/org/sonar/java/model/InternalSyntaxTrivia.java

```diff
@@ -63,7 +63,6 @@ public boolean isLeaf() {
 
   @Override
   public void accept(TreeVisitor visitor) {
-    //FIXME do nothing
```

Comment:
Maybe we should keep the comment, without the `FIXME`, to be explicit that this method does nothing on purpose? Same in `InternalSyntaxToken.java`.